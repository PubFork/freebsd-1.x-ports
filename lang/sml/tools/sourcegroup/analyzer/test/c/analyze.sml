structure Traverse = TraverseFun (Scopes)

structure a = struct
  fun a filename =
    Scopes.prScope (Traverse.traverse (Parse.parseSource filename))
end

