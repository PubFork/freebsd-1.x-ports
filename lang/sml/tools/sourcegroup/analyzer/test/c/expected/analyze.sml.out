source sml "c/analyze.sml"
  import structure Parse Scopes 
  import functor TraverseFun 
  export structure Traverse a ;

